src/color.cc

```diff
@@ -19,6 +19,14 @@ static constexpr const char* color_names[] = {
     "magenta",
     "cyan",
     "white",
+    "brblack",
```

Comment:
we have been moving away from abbreviations in Kakoune source code and in scripts, I suggest to use `bright-something`, fuzzy completion should make it quick to write those (br will match bright-red pretty strongly as it matches 2 word start).

Reply:
`brblack` has widespread use, but I understand moving away from abbreviations. Would `brightblack` (like tmux) suffice?

Reply:
I'd rather have it in two words to make insert completion work well with it.